releasetasks/templates/fennec/version_bump.yml.tmpl

```diff
@@ -0,0 +1,45 @@
+{% set buildername = "release-{}-{}_version_bump".format(branch, product) %}
```

Comment:
Should I merge and address this later when I deal with postrelease builders anyway or should I look into it now?